apps/prechain/src/store.js

```diff
@@ -529,7 +755,7 @@ export const useAccountProposalCandidates = (accountAddress) => {
   return React.useMemo(() => {
     const candidates = Object.values(candidatesById);
     return candidates.filter(
-      (c) => c.proposerId.toLowerCase() === accountAddress.toLowerCase()
+      (c) => c.proposerId?.toLowerCase() === accountAddress.toLowerCase()
```

Comment:
Do we fetch candis without proposer somewhere? Can’t find it

Reply:
when fetching voter activity we get the candidate feedback posts (not full candidates) and use those to eagerly populate the candidates into the state.

Reply:
this was actually not being used (by mistake) before last commit

Reply:
Got it!